spec/spec_helper.cr

```diff
@@ -4,7 +4,21 @@ require "spec"
 require "../src/pluto"
 
 module SpecHelper
-  def self.read_sample(name : String) : String
-    File.read("lib/pluto_samples/#{name}")
+  def self.with_sample(name : String)
```

Comment:
Hm, doesn't ameba enforce a `&` if the method uses `yield` anywhere?

Reply:
Not sure, but CI is happy 😅.